organization/schema.py

```diff
@@ -331,6 +344,70 @@ def mutate(self, info, membership_id, *args, **kwargs):
         return QuitKSGMutation(internal_group_position_membership=membership)
 
 
+class InternalGroupUserHighlightNode(DjangoObjectType):
+    class Meta:
+        model = InternalGroupUserHighlight
+        filter_fields = ["internal_group", "user"]
+        interfaces = (Node,)
+
+    image = graphene.String()
+
+    def resolve_image(self: InternalGroupUserHighlight, info, **kwargs):
+        if self.image:
+            return self.image.url
+        else:
+            return None
+
+    @classmethod
+    def get_node(cls, info, id):
+        return InternalGroupPosition.objects.get(pk=id)
+
+
+class CreateInternalGroupUserHighlightMutation(DjangoCreateMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.add_internalgroupuserhighlight",)
+
+
+class PatchInternalGroupUserHighlightMutation(DjangoPatchMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.change_internalgroupuserhighlight",)
+
+
+class DeleteInternalGroupUserHighlight(DjangoDeleteMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.delete_internalgroupuserhighlight",)
+
+
+class BatchPatchInternalGroupUserHighlightMutation(DjangoBatchPatchMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.change_internalgroupuserhighlight",)
```

Comment:
Will we be using this?

Reply:
no.<img src="https://media4.giphy.com/media/IjD2bKEIiyLfi/giphy.gif"/>